src/ejabberd_tls.erl

```diff
@@ -142,4 +142,4 @@ get_peer_cert(fast_tls, Socket) ->
         {_, error} -> no_peer_cert
     end;
 get_peer_cert(M, S) ->
-    M:get_peer_certificate(S).
\ No newline at end of file
+    M:get_peer_certificate(S).
```

Comment:
`M` -> `Mod` or `Module` ??
`S` ->  `Sock` or `Socket`??

Reply:
I'd rather be consistent with the naming in other functions in this module. It's `M` and `S` everywhere else in most places in this file.

Reply:
So be it :)